accessibility-checker.php

```diff
@@ -187,12 +189,12 @@ function edac_update_database() {
 		require_once ABSPATH . 'wp-admin/includes/upgrade.php';
 		dbDelta( $sql );
 
-	}
+		// Update database version option.
```

Comment:
Moved this to inside the if statement so it will only make the db call if the version has changed.